app/main.py

```diff
@@ -15,5 +15,12 @@
     "Sergio": 3,
 }
 collection_of_coins = {1, 2, 25}
+sorted_variables = {"mutable" : [], "immutable" : []}
+all = [lucky_number, pi, one_is_a_prime_number, name, my_favourite_films, profile_info, marks, collection_of_coins]
```

Comment:
It's a good practice to avoid using built-in function names as variable names to avoid confusion and potential bugs. 'all' is a built-in function in python, consider changing this variable name.